src/Symfony/Component/Cache/Adapter/FilesystemAdapter.php

```diff
@@ -126,7 +126,7 @@ protected function doDelete(array $ids)
     protected function doSave(array $values, $lifetime)
     {
         $ok = true;
-        $expiresAt = $lifetime ? time() + $lifetime : PHP_INT_MAX;
+        $expiresAt = time() + ($lifetime ?: 31557600); // 31557600s = 1 year
```

Comment:
How about defining a constant instead of adding the comment? Something like `self::YEAR` would be self explanatory.